build_calico_kubernetes/requirements.txt

```diff
@@ -1,6 +1,7 @@
 PyInstaller==2.1
-git+https://github.com/projectcalico/libcalico.git
+git+https://github.com/projectcalico/libcalico.git@master
```

Comment:
We don't want to be tracking master here in our dependencies - we should track a specific version of libcalico that we know works.